vendor/github.com/Cray-HPE/shasta-cfg/customizations.yaml

```diff
@@ -565,20 +565,6 @@ spec:
         storage_shared_size_cray_tftp: 5Gi
         nmn_tftp_service_ip: '{{ network.netstaticips.nmn_tftp }}'
         hmn_tftp_service_ip: '{{ network.netstaticips.hmn_tftp }}'
-      cray-uas-mgr:
```

Comment:
@keopp-hpe Please propose this change separately for https://github.com/Cray-HPE/shasta-cfg, this is a vendored repo.